pkg/block/fetcher.go

```diff
@@ -566,4 +573,51 @@ func (f *ConsistencyDelayMetaFilter) Filter(metas map[ulid.ULID]*metadata.Meta,
 			delete(metas, id)
 		}
 	}
+
+	return nil
+}
+
+// IgnoreDeletionMarkFilter is a MetaFetcher that filters blocks that are marked for deletion; delay duration before current time.
```

Comment:
It's not a MetaFetcher :cry:

Reply:
> delay duration before the current time.

This is out of nowhere, what about:
```suggestion
// IgnoreDeletionMarkFilter is a filter that filters out the blocks that are marked for deletion after a given delay.
```

Reply:
I was having trouble correctly wording it, the semicolon was looking bad. The suggestion that you made is really better.